cdap-ui/app/cdap/services/GuidedTour/index.ts

```diff
@@ -23,11 +23,11 @@ interface IStepButton {
   classes?: string;
   action: () => void;
 }
-
+// Setting the interface for text and title to 'any' because unsure about the return type of T.translate
```

Comment:
@elfenheart the return type of T.translate is
```
declare module 'i18n-react' {
  import React = require('react');
  export class MDText {
      texts: any;
      constructor(texts: any);
      setTexts(texts: any): void;
      notFound: string;
      interpolate(exp: string, vars: any): any;
      format(value: string, vars?: any): React.ReactNode;
      translate(key: string, options?: any): React.ReactNode;
      factory(tag: string): (props: any) => React.ComponentElement<any, React.Component<any, {} | void>>;
      p: (props: any) => React.ComponentElement<any, React.Component<any, {} | void>>;
      span: (props: any) => React.ComponentElement<any, React.Component<any, {} | void>>;
      li: (props: any) => React.ComponentElement<any, React.Component<any, {} | void>>;
      div: (props: any) => React.ComponentElement<any, React.Component<any, {} | void>>;
      button: (props: any) => React.ComponentElement<any, React.Component<any, {} | void>>;
      a: (props: any) => React.ComponentElement<any, React.Component<any, {} | void>>;
      text: (props: any) => React.ComponentElement<any, React.Component<any, {} | void>>;
  }
  var singleton: MDText;
  export default singleton;
}

```

vscode should show this. The problem however is you need to convert the return type to string using `toString` otherwise ts will complain I believe.